pasion_preciouskaye/scripts.js

```diff
@@ -64,7 +64,7 @@ function searchCountry() {
   var countryName = document.getElementById('countryInput').value.trim();
```

Comment:
It looks like you forgot to update your js file. IDs should be written in snake case. All lower case separated by underscore. Also, always use "let" keyword in declaring variables over the "var" keyword. 

```suggestion
  let countryName = document.getElementById('country_input').value.trim();
```

For reference, please click on the link provided below to access further information. 
(https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5)

(https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g25d183194fb_0_0)